Sources/Core/CustomStruct.swift

```diff
@@ -0,0 +1,143 @@
+  }
+
+  // Variable that is a class
+  public var customClassVariable: Employee
+
+  // Variable that is an enum
+  public var enumVariable: EmployeePayroll
+
+  // Variable that is an array of enum
+  public var enumArrayVariable: [EmployeePayroll]
+
+  private var privateStringVariable: String
+
+  // Public Variable
+  public var stringVariable: String
+
+  public var structVariable: StructWithGeneratedInitializer
+
+  // Computed internal var
+  var computedInternalDateVariable: Date {
+    return Date()
+  }
+
+  public init(
+    stringConstant: String,
+    stringArray: [String],
+    stringArrayWithDefaultValue: [String],
+    customClassVariable: Employee,
+    enumVariable: EmployeePayroll,
+    enumArrayVariable: [EmployeePayroll],
+    stringVariable: String,
+    structVariable: StructWithGeneratedInitializer
+    ) {
+    self.stringConstant = stringConstant
+    self.stringArray = stringArray
+    self.stringArrayWithDefaultValue = stringArrayWithDefaultValue
+    self.stringVariable = stringVariable
+    self.customClassVariable = customClassVariable
+    self.enumVariable = enumVariable
+    self.privateStringVariable = "secret"
+    self.structVariable = structVariable
+    self.enumArrayVariable = enumArrayVariable
+  }
+
+  init(
+    stringConstant: String,
+    stringArray: [String],
+    stringArrayWithDefaultValue: [String],
+    customClassVariable: Employee,
+    enumVariable: EmployeePayroll,
+    enumArrayVariable: [EmployeePayroll],
+    privateStringVariable: String,
+    stringVariable: String,
+    structVariable: StructWithGeneratedInitializer
+    ) {
+    self.stringConstant = stringConstant
+    self.stringArray = stringArray
+    self.stringArrayWithDefaultValue = stringArrayWithDefaultValue
+    self.privateStringVariable = privateStringVariable
+    self.stringVariable = stringVariable
+    self.customClassVariable = customClassVariable
+    self.enumVariable = enumVariable
+    self.structVariable = structVariable
+    self.enumArrayVariable = enumArrayVariable
+  }
+
+  public func methodWithNoParametersReturnsNativeType() -> Date {
+    return computedInternalDateVariable.addingTimeInterval(Double(60 * 60 * 60))
+  }
+
+  public func methodWithParameters(
+    byHours hours: Int,
+    minutes: Int,
+    _ seconds: Int) {
+    print("\(hours), \(minutes), \(seconds)")
+  }
+
+  public static func staticMethodWithParameters(
+    byHours hours: Int,
+    minutes: Int,
+    _ seconds: Int) {
+    print("\(hours), \(minutes), \(seconds)")
+  }
+
+  public func methodWithNoParametersReturnsClass() -> Employee {
+    return customClassVariable
+  }
+
+  public mutating func mutatingMethod(newColors: StructWithGeneratedInitializer) {
+    self.structVariable = newColors
+  }
+  // Public Method that takes an enum
+
+  // Public Method that returns an enum
+
+  // Public Method that returns a bridged type
+}
+
+// Phantom type to mark types that may be bridged to objc
+protocol ObjCBridgeable {}
```

Comment:
Shouldn't this be in its own file? Something like:

```swift
// ObjCBridgeable.swift

// MARK: - Protocol

protocol ObjCBridgeable {}

// MARK: - Bridgeable

extension CustomStruct: ObjCBridgeable {}
extension Employee: ObjCBridgeable {}
extension EmployeeClassification: ObjCBridgeable {}
extension EmployeePayroll: ObjCBridgeable {}
extension Graph: ObjCBridgeable {}
extension Parser: ObjCBridgeable {}
extension StructWithGeneratedInitializer: ObjCBridgeable {}
```

Reply:
I like that idea. Making the change.